### PR TITLE
Di91 feedback form

### PR DESCRIPTION
**Pull request by JPrevost**

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
```
Comment (frrrances) [inline app/views/layouts/mailer.html.erb]: sad but true. :'(

File: test/fixtures/feedback_mailer/feedback_email [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  <p>This is an important message!</p>
+
+  <p>Contact Email: yo@example.com</p>
+
+  <p>Client IP: 0.0.0.0</p>
+
+  <p>Originating page: http://example.com/stuff</p>
+
+  <p>User Agent: Netscape 4.0</p>
```
Comment (frrrances) [inline test/fixtures/feedback_mailer/feedback_email]: best browser of all time!

File: test/mailers/previews/feedback_mailer_preview.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Preview all emails at http://localhost:3000/rails/mailers/feedback_mailer
+class FeedbackMailerPreview < ActionMailer::Preview
+  def feedback_email
+    msg = 'This is a message. <p>It is important</p>'
```
Comment (frrrances) [inline test/mailers/previews/feedback_mailer_preview.rb]: are we allowing html in feedback?

File: app/assets/stylesheets/feedback.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  height: 10em;
+}
+
+.bento-field-textarea.error, .bento-field-textarea.error:focus {
+  border: 1px solid red;
+}
+
+label.error {
+  color: red;
+}
```
Comment (frrrances) [inline app/assets/stylesheets/feedback.scss]: I assume I can adjust these as i like later...

Comment (JPrevost) [inline app/assets/stylesheets/feedback.scss]: No.

File: app/views/feedback/submit.html.erb [hunk 5]
```diff
@@ -0,0 +1 @@
+<p>Feedback Submited</p>
```
Comment (JPrevost) [inline app/views/feedback/submit.html.erb]: Well, I was going for `smited` but `submitted` does make more sense. Thx!

File: test/mailers/previews/feedback_mailer_preview.rb (hunk 3 above)
Comment (JPrevost) [inline test/mailers/previews/feedback_mailer_preview.rb]: Sort of. It accepts anything but it escapes when sending the email so it shouldn't render in your email client. You'll see that they submitted HTML, but it'll be like ```<p>hi!</p>```

**Pull request merged**